### ISSUE TITLE
paste of long code to ESP32 REPL leads to corrupted data transfer

### ISSUE THREAD

**Issue by JohnMTu** [JetBrains/intellij-micropython#206]
Hello,

when trying to paste few KB long code via REPL (using ctrl+e) on ESP32, pasted data are often corrupted.
It depends on ESP32 FW version of how big the file can be to get errors.

As workaround i have to cut it to smaller pieces and paste each individually.
I guess some kind of few millis delay after each few hundreds of bytes would help.
Is there a place where i could to configure/hack it?

thanks

**Comment by Elizaveta239** [comment]
Duplicate of #142
